source/served/commands/complete.d

```diff
@@ -563,7 +568,11 @@ CompletionList provideDietSourceComplete(TextDocumentPositionParams params,
 			ret.label = a.text;
 			ret.kind = a.type.mapToCompletionItemKind.opt;
 			if (a.definition.length)
+			{
 				ret.detail = a.definition.opt;
+				if (capabilities.textDocument.completion.completionItem.labelDetailsSupport)
+					ret.labelDetails = CompletionItemLabelDetails(ret.detail);
```

Comment:
diet might need some testing and the change here could either be left out or be implemented properly